modules/ROOT/pages/constraints/syntax.adoc

```diff
@@ -16,6 +16,7 @@ This means its default behavior is to throw an error if an attempt is made to cr
 With the `IF NOT EXISTS` flag, no error is thrown and nothing happens should a constraint with the same name or same schema and constraint type already exist.
 It may still throw an error if conflicting data, indexes, or constraints exist.
 Examples of this are nodes with missing properties, indexes with the same name, or constraints with same schema but a different conflicting constraint type.
+In the case when nothing happens an informational notification will be returned showing the pre-existing constraint blocking the creation.
```

Comment:
```suggestion
As of Neo4j 5.17, in case nothing happens an informational notification is returned showing the existing constraint which blocks the creation.
```

multiple occurrences